.github/workflows/test-license-scan.yml

```
@@ -0,0 +1,16 @@
+name: Test bottlerocket-license-scan
+on:
+    pull_request:
+        branches:
+            - "*"
+        paths:
+            - "license-scan/**"
+jobs:
+    build:
+        runs-on: ubuntu-latest
```

I'm not sure we can use `ubuntu-latest` as a runner; it might need to be shared from the org level.

I noticed that the only core kit workflow to use this runner - [golangci-lint.yaml](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/develop/.github/workflows/golangci-lint.yaml) doesn't seem to have triggered for a [recent PR](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/146) - so this might be a new change.

Try changing it to this and see if the workflow runs:
```suggestion
        runs-on:
            group: bottlerocket
            labels: bottlerocket_ubuntu-latest_8-core
```